WLC/benchmark.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from WLC.image_processing.camera import Camera
+from WLC.image_processing.preprocessor import Preprocessor
+
+import editdistance
+
+
+def _get_expected_code(file_name):
+    file_name = file_name.replace('images', 'annotations')
+    file_name = re.sub(r"\..*", ".txt", file_name)
+    file_name = Camera().get_full_path(file_name)
```

Comment:
This seems very fragile, a rename would break the replace

Reply:
Yeah but not sure thats the better solution. We need some standard.